src/Analysis/Engine/Impl/Values/BuiltinInstanceInfo.cs

```diff
@@ -33,6 +33,7 @@ public BuiltinInstanceInfo(BuiltinClassInfo klass)
         IBuiltinClassInfo IBuiltinInstanceInfo.ClassInfo => ClassInfo;
         public BuiltinClassInfo ClassInfo => _klass;
 
+        public override string Name => _klass.Name;
```

Comment:
Do we need similar change for `InstanceInfo`?

Reply:
Won't hurt, I'll add them.